jsk_recognition_utils/requirements.txt

```diff
@@ -0,0 +1,2 @@
+chainer>=4.2.0
+scikit-image>=0.13.1
```

Comment:
0.13.1 is right?

Reply:
`catkin run_tests` passed with scikit-image>=0.12.1, so updated.